gui.cpp

```diff
@@ -785,7 +785,7 @@ void UI::minimize(tgui::ChildWindow::Ptr win) {
 
 void UI::changeServerAdd(sf::String addr) { //Quickfix for issue #33
 	std::string serveraddr = addr.toAnsiString();
-	if (serveraddr.back() != '.' || serveraddr.size() != 0)
+	if (!(serveraddr.back() == '.' || serveraddr.size() == 0))
```

Comment:
What would make things more clear is if there was a "Merging with origin/master" commit to show the workflow.